src/components/Organisms/Header/HeaderNav/HeaderNav.js

```diff
@@ -0,0 +1,214 @@
+import React, { useState, useEffect, useCallback } from 'react';
```

Comment:
Updates required for HeaderNav2025 meant refactoring the _preexisting_ one; still works in the same way.